setup.py

```diff
@@ -68,6 +68,8 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
```

Comment:
Might be a good idea to just keep `Python::3`, we've started to do that other project.  Not sure though, don't really care.